src/TermStoreInstaller.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Wikibase\TermStore;
+
+interface TermStoreInstaller {
+
+	/**
+	 * @throws TermStoreException
```

Comment:
I would remove interface exceptions entirely .. will do that in a separate PR to discuss there